common/go-utils/src/inv.rs

```diff
@@ -23,6 +25,18 @@ pub struct Artifact {
     pub sha_checksum: String,
 }
 
+impl Hash for Artifact {
```

Comment:
this `impl` should have a unit test

Reply:
Added in https://github.com/heroku/buildpacks-go/pull/230/commits/36712ee15c89568572e94afd4acd17e2cacc6de3